### PR TITLE
Update package Libsolv to version 0.7.27

### PR DESCRIPTION
**Pull request by Alberto-Perez-Guevara**
Changes to be committed:
	modified:   ../SPECS/libsolv/libsolv.signatures.json
	modified:   ../SPECS/libsolv/libsolv.spec
	modified:   ../cgmanifest.json
	modified:   resources/manifests/package/pkggen_core_aarch64.txt
	modified:   resources/manifests/package/pkggen_core_x86_64.txt
	modified:   resources/manifests/package/toolchain_aarch64.txt
	modified:   resources/manifests/package/toolchain_x86_64.txt

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

Update package Libsolv to version 0.7.27

###### Change Log  

 Changes to be committed:
	modified:   ../SPECS/libsolv/libsolv.signatures.json
	modified:   ../SPECS/libsolv/libsolv.spec
	modified:   ../cgmanifest.json
	modified:   resources/manifests/package/pkggen_core_aarch64.txt
	modified:   resources/manifests/package/pkggen_core_x86_64.txt
	modified:   resources/manifests/package/toolchain_aarch64.txt
	modified:   resources/manifests/package/toolchain_x86_64.txt

###### Does this affect the toolchain?  

YES

###### Test Methodology

- Local build
- Pipeline build id: xxxx

### CODE REVIEW

File: cgmanifest.json [hunk 1]
```diff
@@ -10941,8 +10941,8 @@
         "type": "other",
         "other": {
           "name": "libsolv",
-          "version": "0.7.24",
-          "downloadUrl": "https://github.com/openSUSE/libsolv/archive/0.7.24/libsolv-0.7.24.tar.gz"
+          "version": "0.7.27",
+          "downloadUrl": "https://src.fedoraproject.org/lookaside/pkgs/libsolv/libsolv-0.7.27.tar.gz/sha512/5f4de944114cebbcf35a993e5f969c3b7bf3b8dbc5dabab38014bd2d1ab83b524d1ed5b7fbed4b2f751c1f8043613f9e88cf96b14b121362655af1805d238b60/libsolv-0.7.27.tar.gz"
```
Comment (mfrw) [inline cgmanifest.json:10945]: Why are we moving away from `GitHub` source ?

Comment (Alberto-Perez-Guevara) [reply cgmanifest.json:10925]: the original source is no longer available, from the original source you can only download source files but I couldn't find any tar balls

File: SPECS/libsolv/libsolv.spec [hunk 2]
```diff
@@ -1,10 +1,10 @@
 Summary:        A free package dependency solver
 Name:           libsolv
-Version:        0.7.24
+Version:        0.7.27
 Release:        1%{?dist}
 License:        BSD
 URL:            https://github.com/openSUSE/libsolv
-Source0:        https://github.com/openSUSE/libsolv/archive/%{version}/%{name}-%{version}.tar.gz
+Source0:        https://src.fedoraproject.org/lookaside/pkgs/libsolv/%{name}-%{version}.tar.gz/sha512/5f4de944114cebbcf35a993e5f969c3b7bf3b8dbc5dabab38014bd2d1ab83b524d1ed5b7fbed4b2f751c1f8043613f9e88cf96b14b121362655af1805d238b60/%{name}-%{version}.tar.gz
```
Comment (mfrw) [inline SPECS/libsolv/libsolv.spec:7]: I think if we look at the tags, i could find a tar.gz url :)
URL: https://github.com/openSUSE/libsolv/archive/refs/tags/0.7.28.tar.gz

Comment (Alberto-Perez-Guevara) [reply SPECS/libsolv/libsolv.spec:7]: Thanks and actually its a newer revision than the one I have right now I have 0.7.27 and seems that 0.7.28 is out I think I'll abandon this review and start a new one with the latest revision

**Comment by Alberto-Perez-Guevara** [comment]
newer revision than the one I have right now I have 0.7.27 and seems that 0.7.28 is out I think I'll abandon this review and start a new one with the latest revision

**Pull request closed without merging**